tabsie/markDown.Rmd.orig

```diff
@@ -1,77 +0,0 @@
----
```

Comment:
This file was a merge artifact (identical to markDown.Rmd) that I somehow created when splitting out TABSIE into its own project. Deleted.